src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -639,12 +639,11 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
                 }
             } else if (port_type == "out") {
                 const auto& node_outputs = indexed_from_named(ng_op_map[operation_name]);
-                FRONT_END_GENERAL_CHECK(node_outputs.size() > port_index,
-                                        "Output port with index " + std::to_string(port_index) + " of " +
-                                            operation_name + "node specified as custom output does not exist");
-                auto result_node = std::make_shared<ov::opset8::Result>(node_outputs[port_index]);
-                result_node->set_friendly_name(model_output_name);
-                results.push_back(result_node);
+                if (node_outputs.size() > port_index) {
+                    auto result_node = std::make_shared<ov::opset8::Result>(node_outputs[port_index]);
+                    result_node->set_friendly_name(model_output_name);
+                    results.push_back(result_node);
+                }
```

Comment:
why do we need this change?

Reply:
It is needed as we set names with ":0" in `InputModel`. 
`model_output` with name that ends with ":0" is processed in `port_type == "out"` branch (otherwise it would be processed in `port_type == "none"` branch), with `port_index==0`. 
It has `node_outputs.size() == 0`, as it is output, so I removed the check, which checks `node_outputs.size() > port_index`. 

We still create new `Result` here if it is not terminating node, but allow places with ":0" to not have any outputs.